GraphWebhooks/Helpers/GraphHttpClient.cs

```diff
@@ -0,0 +1,72 @@
+﻿
+namespace GraphWebhooks.Helpers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Threading.Tasks;
+
+    public class GraphHttpClient
+    {
+        private static readonly int MaxRetryAttempts = 5;
+
+        private string accessToken = null;
+
+        public GraphHttpClient(string accessToken)
+        {
+            this.accessToken = accessToken;
+        }
+
+        // Send HTTP request implemenation with retries on transient failures.
```

Comment:
Minor nit: typo in the comment:  implementation

Reply:
Fixed. Thanks for the minute observation.